src/lib.rs

```diff
@@ -92,9 +92,21 @@ impl Default for Type {
     }
 }
 
-/// Builder for the generator.
-#[derive(Default, Clone, Debug, PartialEq)]
-pub struct Builder<F, V>
+/// This is the error type for the `Builder::build` function.
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+pub enum ConfigurationError {
```

Comment:
Hmm... When I was first writing the code I didn't go with custom error as all the errors were caused by missuse of the API and thus programming errors which means that there isn't anything meaningful to do if code triggers them.
Maybe if one calculates radius via other code before setting it via builder then it could have use?
So maybe custom error is better anyways.